src/compile_torture_tests.py

```diff
@@ -82,7 +89,7 @@ def run(c, cxx, testsuite, sysroot_dir, fails, out, config='wasm'):
   return result
 
 
-def getargs():
```

Comment:
I guess I'm just used to seeing 'main' do the argument handling since nobody else needs to do this.
 
If I do it as you suggest main becomes two line function and the only caller to getargs()... so I guess like it better this way but its not a huge deal.